src/v3/src/transformer/profile/transformEnrollProfile.ts

```diff
@@ -74,14 +85,26 @@ export const transformEnrollProfile: IdxStepTransformer = ({ transaction, formBa
     },
   };
 
+  if (isAlternateView) {
```

Comment:
see: https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/EnrollProfileView.js#L9-L23

Reply:
Can we try to file a dependency against the home team to fix this from the backend side?

Reply:
Done! Had a conversation with them last week about it. 

https://oktainc.atlassian.net/browse/OKTA-539666

https://okta.slack.com/archives/C9AJXB5GC/p1665149067212879

Reply:
Starting this discussion with the feature team: 

https://okta.slack.com/archives/C029NJQTTC5/p1665422645575769 

Will update with the ticket once I get a response from them.

Reply:
Spoke to the team, and created a ticket for them: 
https://oktainc.atlassian.net/browse/OKTA-540383

Discussion: 
https://okta.slack.com/archives/C029NJQTTC5/p1665422645575769